src/pages/basic_data/app_admin/central_import/CentralImportCandidateMerge.tsx

```diff
@@ -129,15 +130,19 @@ export const CentralImportCandidateMerge = () => {
           <Helmet>
             <title>{t('basic_data.central_import.central_import')}</title>
           </Helmet>
+          <Typography component="h1" sx={visuallyHidden}>
```

Comment:
Her er title og h1 ulik. Bevisst?

Tenker det kan være naturlig å vise h1 her også, da det ser litt ut som at noe mangler der i dag?
![bilde](https://github.com/user-attachments/assets/d1ba4252-a16d-44ae-b10b-c72d34be844c)

```suggestion
            <title>{t('basic_data.central_import.merge_candidate.merge')}</title>
          </Helmet>
          <Typography variant="h1">
```